cmd/main.go

```diff
@@ -70,6 +72,23 @@ func StartSecurityPolicyController(mgr ctrl.Manager, commonService common.Servic
 	}
 }
 
+func StartSubnetPortController(mgr ctrl.Manager, commonService common.Service) {
+	subnetPortReconcile := &subnetport_controller.SubnetPortReconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
+	}
+	if subnetPortService, err := subnetport_service.InitializeSubnetPort(commonService); err != nil {
```

Comment:
This should be updated if this pr merged.
https://github.com/vmware-tanzu/nsx-operator/pull/160/files#diff-56a01862f16c1cc9248df82cc17bcd7adcb1fe24442da3fb078cfabee9b9c7e9

Reply:
Thanks for the reminder. If the patch for IPPool was merged before this one, the build will fail so we can get the notification and update the codes.